Makefile

```diff
@@ -78,5 +78,8 @@ lint:
 generate:
 	go generate ./...
 
+prerequisite-check:
+	@bash tools/prerequisite-check.sh
```

Comment:
Can we add a path to skip this check like `make PREREQUISITE_CHECK=0` or another way? It is useful for an expert who wants to run this program not on a recommended environment.

Reply:
Sure, thanks for the suggestion. I'll define `SKIP_PREREQUISITE_CHECK` to do prerequisite check by default.